app/views/courses/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<p id="notice"><%= notice %></p>
```

Comment:
The show route is going to end up rendering the current landing page (MainController.index) probably so we won't actually need a separate show view. I would remove this file as well.